mitodl/mitxonline
# Fixing Paid tag display on checkout page

#### Pre-Flight checklist


#### What are the relevant tickets?
#501 

#### What's this PR do?
Fixes #501 

#### How should this be manually tested?
Add an item to the cart and proceed to the Checkout page - you should not see the Paid tag. 

Go to the Order History page and choose an order that has not been refunded. You should see the Paid tag. Then, choose an order that has been refunded. You should see the Refund Applied tag and refund amount.
